hydroshare/local_settings.template

```diff
@@ -138,12 +138,7 @@ POSTGIS_VERSION = (2, 1, 1)
 HS_BAGIT_PATH = 'bags'
 HS_BAGIT_POSTFIX = 'zip'
 
-HS_BAGIT_README_FILE_WITH_PATH = 'docs/bagit/readme.txt'
-
-# crossref login credential for resource publication
-USE_CROSSREF_TEST = True
-CROSSREF_LOGIN_ID = ''
-CROSSREF_LOGIN_PWD = ''
+HS_BAGIT_README_FILE_WITH_PATH = 'docs/bagit/readme.txt
```

Comment:
> I’m seeing an issue on the resource landing page on branch rm_crossref. The “Add files” button doesn’t do anything (no dialog, no request). Drag-and-drop also seems inactive. Could you take a look or point me to what might have changed, wondering if I broke something there.

@MSDrao here is the first issue I see with this PR, using a simple check of the logs:

<img width="799" height="907" alt="Image" src="https://github.com/user-attachments/assets/8251c5bd-9770-4f19-ac6e-3fa20bb64176" />